Structural and line edit on concepts/hypertables (#810)
timescale/docs: timescaledb/overview/core-concepts/hypertables-and-chunks/index.md

handle time-series data. Anything you can do with a regular PostgreSQL table,
you can do with a hypertable. In addition, you get the benefits of improved
performance and user experience for time-series data.

With hypertables, TimescaleDB makes it easy to [improve insert and query
performance][performance-benchmark] by partitioning time-series data on its time
parameter. Behind the scenes, the database performs the work of setting up and
maintaining the hypertable's partitions. Meanwhile, you insert and query your
data as if it all lives in a single, regular PostgreSQL table.

A hypertable's behind-the-scenes partitions, called chunks, also make other
features of TimescaleDB possible. These include continuous aggregates,
compression, retention policies, and more.

Learn more about:
*   The [benefits of hypertables][hypertable-benefits] for handling time-series
    data
*   [Hypertable architecture][hypertable-architecture], and how hypertables and
    chunks work behind the scenes
*   [How to work with hypertables][hypertable-how-to]

## Hypertables compared to partitioning in regular PostgreSQL
If you use regular PostgreSQL, you can also partition your time-series data by

> writing the partitioning logic yourself. But handling child tables, constraints,

Author: @Loquacity is this better?

chunk indexes, and other details gets complex. And that's before you get into
more advanced features such as compression, continuous aggregates, and retention
policies.

TimescaleDB handles all this for you so you can focus on your application.

## When to use a hypertable in TimescaleDB
You can have both hypertables and regular PostgreSQL tables in the same
database. Choose a hypertable for time-series data, and choose a regular
PostgreSQL table for relational data.

For example, you might have:
*   A hypertable to record sensor readings over time, and a regular table
    to record sensor location and other metadata
*   A hypertable to record stock asset prices over time, and a regular table to
    record ticker symbols and names for each asset
*   A hypertable to record truck GPS coordinates over time, and a regular table
    to record the make, model, and age of each truck

> [hypertable-architecture]: /overview/core-concepts/hypertables-and-chunks/hypertable-architecture/
> [hypertable-benefits]: /overview/core-concepts/hypertables-and-chunks/hypertables-and-chunks-benefits/
> [hypertable-how-to]: /how-to-guides/hypertables/

Author: ```suggestion
[hypertable-architecture]: /timescaledb/:currentVersion:/overview/core-concepts/hypertables-and-chunks/hypertable-architecture/
[hypertable-benefits]: /timescaledb/:currentVersion:/overview/core-concepts/hypertables-and-chunks/hypertables-and-chunks-benefits/
[hypertable-how-to]: /how-to-guides/hypertables/
```

For example, assume each chunk contains 1 day's worth of data. For each day
for which you have data, TimescaleDB creates a chunk. For each row of data, it
looks at the time column and inserts the data into the corresponding chunk. All
rows with time values from the same day are inserted into the same chunk. Rows
belonging to different days are inserted into different chunks.

<img class="main-content__illustration"
src="https://s3.amazonaws.com/assets.timescale.com/docs/images/getting-started/hypertables-chunks.png"
alt="Rows are inserted into hypertable chunks based on their time values." />

This happens behind the scenes. You run regular inserts and queries to your
database, and TimescaleDB automatically handles the partitioning.

In addition to time, you can also partition a hypertable by space parameters.
For more information, see the section on [time-and-space
partitioning](#hypertables-can-also-be-partitioned-by-space).

<highlight type="note">
This section uses the example of 1-day chunks to explain how hypertables work.
The default chunk time interval is actually 7 days, and you can change it to
optimize partitioning for your data. To learn about best practices for chunk 

> time intervals, see the documentation on [chunk sizing](FIXME).

Author: ```suggestion
time intervals, see the documentation on [chunk sizing](timescaledb/latest/how-to-guides/hypertables/best-practices/#time-intervals).
```

# Hypertables
Hypertables are PostgreSQL tables with special features that make it easy to
handle time-series data. Anything you can do with a regular PostgreSQL table,
you can do with a hypertable. In addition, you get the benefits of improved

> performance and user experience for time series.

Reviewer: ```suggestion
performance and user experience for time series data.
```

*   A hypertable to record truck GPS coordinates over time, and a regular table
    to record the make, model, and age of each truck

## Why can't you partition your time-series data yourself?

> You can! You can plan and write all the partitioning logic yourself on regular

Reviewer: Not sure I like this construction 😕
